app/views/user_mailer/claim_notification.html.erb

```diff
@@ -25,9 +25,7 @@
     <% end %>
   </ul>
 
-  <p><%= t(".redirects.html", negation: style_bold(t(".redirects.negation"))) %></p>
-
-  <p><%= t(".update_redirect.html", contact_open_doors_link: opendoors_link(t(".update_redirect.contact_open_doors"))) %></p>
+  <p><%= t(".redirects.html", negation: style_bold(t(".redirects.negation")), faq_link: style_link(t(".html.faq"), archive_faqs_url)) %></p>
```

Comment:
```suggestion
  <p><%= t(".redirects.html", negation: style_bold(t(".redirects.negation")), faq_link: style_link(t(".redirects.faq"), archive_faqs_url)) %></p>
```
This is the more modern style of where to put the link captions and matches the rest of the email